aws/logs_monitoring/parsing.py

```diff
@@ -57,7 +57,7 @@
 rds_regex = re.compile("/aws/rds/(instance|cluster)/(?P<host>[^/]+)/(?P<name>[^/]+)")
 
 cloudtrail_regex = re.compile(
-    "\d+_CloudTrail_\w{2}-\w{4,9}-\d_\d{8}T\d{4}Z.+.json.gz$", re.I
+    "\d+_CloudTrail_\w{2}(|-gov)-\w{4,9}-\d_\d{8}T\d{4}Z.+.json.gz$", re.I
```

Comment:
Do we need to do `-cn` as well?

Reply:
Yep, added. Also added support for cloudtrail digest.